### PR TITLE
Add table of content for instant meilisearch customization

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,13 @@ const searchClient = instantMeiliSearch(
 
 InstantMeilisearch offers some options you can set to further fit your needs.
 
+### Customization
+
+- [`placeholderSearch`]((#placeholder-search)): Enable or disable placeholder search (default: `true`).
```
Comment (mdubus) [inline README.md:83]: You have double parenthesis, this breaks the link :D

**Review by mdubus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bidoubiwa) [reply README.md:81]: Oh thanks!

**Review by alallema — APPROVED** [review]
LGTM!

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
bors mergebors merge

**Pull request merged**